src/work-allocation-2/components/find-person/find-person.component.ts

```diff
@@ -44,7 +46,7 @@ export class FindPersonComponent implements OnInit {
   }
 
   public filter(searchTerm: string): Observable<Person[]> {
-    if (searchTerm && searchTerm.length > this.minSearchCharacters) {
+    if (searchTerm && searchTerm.length >= this.minSearchCharacters) {
```

Comment:
This may be why 2 characters start the search instead of 3 - If we want >= then we need to reset minSearchCharacters to 3

Reply:
I now set the this.minSearchCharacters=3, that make it more sense for minSearchCharacters, this code will keep as >=